sale_double_validation/models/sale.py

```diff
@@ -38,10 +46,21 @@ def is_to_approve(self):
     @api.model
     def create(self, vals):
         obj = super().create(vals)
-        if obj.is_to_approve():
-            obj.state = 'to_approve'
+        if not obj.is_to_approve():
```

Comment:
Why was this reversed?

Reply:
we want to_approve to be the default state and move to draft if it doesn't need to be approved. Reason is that in draft state there are possibilities to move it to a different state directly from creation form if we have draft as the default